apps/cowswap-frontend/src/legacy/state/application/initialState.ts

```diff
@@ -1,17 +1,17 @@
-import { localWarning } from './localWarning'
+// import { localWarning } from './localWarning'
 import { ApplicationState } from './reducer'
 
 const popupList: ApplicationState['popupList'] = []
 
-if (localWarning) {
-  popupList.push({
-    key: 'localWarning',
-    show: true,
-    removeAfterMs: null,
-    content: {
-      warning: localWarning,
-    },
-  })
-}
+// if (localWarning) {
+//   popupList.push({
+//     key: 'localWarning',
+//     show: true,
+//     removeAfterMs: null,
+//     content: {
+//       warning: localWarning,
+//     },
+//   })
+// }
```

Comment:
i just coment out to bring it back if necessary. We just don't need to upload to IPFS anymore, but is a pity to delete this.

HAppy to kill it if reviewers desire so